##
#
# Defines the initial state of the controller
#
##
enabled: true

##
#
# Control rules for each control mode
# Possible values for control rules are
# - x
# - y
# - z
# - roll
# - pitch
# - yaw
# - surge
# - sway
# - heave
#

control_modes:
  ##
  # PID gains for each degree of freedom
  ##
  heading:
    yaw:      {p: 20.0,  i: 0.5,   d: 30.0,  i_max: 20, i_min: -20}

  flight:
    pitch:    {p: 5.0,    i: 1.0,   d: 2.0,   i_max: 10, i_min: -10}
    yaw:      {p: 5.0,    i: 0.5,   d: 30.0,  i_max: 20, i_min: -20}
    surge:    {p: 10.0,   i: 3.0,   d: 5.0,   i_max: 10, i_min: -10}

  hold:
    x:        {p: 1.0,    i: 0.5,   d: 1.0,   i_max: 10, i_min: -10}
    y:        {p: 1.0,    i: 0.5 ,  d: 1.0,   i_max: 10, i_min: -10}
    z:        {p: 1.0,    i: 1.0 ,  d: 10.0,  i_max: 10, i_min: -10}
    pitch:    {p: 5.0,    i: 1.0,   d: 2.0,   i_max: 10, i_min: -10}
    yaw:      {p: 5.0,    i: 0.5,   d: 30.0,  i_max: 20, i_min: -20}

  kill: false